helm-chart/renku/templates/data-service/rbac.yaml

```diff
@@ -80,6 +80,18 @@ rules:
       - list
       - get
       - watch
+  - apiGroups:
+      - shipwright.io
+    resources:
+      - buildruns
+    verbs:
+      - create
+      - update
+      - delete
+      - patch
+      - list
+      - get
+      - watch
```

Comment:
Does this blow up if the cluster does not have the buildrun CRDs installed?

Same for the notebooks role changes below.

Reply:
I just tried to create a role (with a role binding and service account) just like this in a bare kind cluster. It worked. So this will not blow things up if shipwright or tekton is not installed.